django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -148,12 +148,35 @@
     border-bottom: none;
 }
 
-#changelist-filter h3 {
+#changelist-filter details summary {
+    cursor: pointer;
     font-weight: 400;
     padding: 0 15px;
     margin-bottom: 10px;
 }
 
+#changelist-filter details summary > * {
+    display: inline;
+}
+
+#changelist-filter details > summary {
+    list-style-type: none;
+}
+
+#changelist-filter details > summary::-webkit-details-marker {
+    display: none;
+}
+
+#changelist-filter details > summary::before {
+    content: '🔽';
+}
+
+#changelist-filter details[open] > summary::before {
+    content: '🔽';
+    display: inline-block;
+    transform: rotate(270deg);
```

Comment:
In Chrome, Linux the icon is not showed. I guess because we don't load the font and is not in my OS?